### DISCUSSION TITLE
Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 15844 path $

### DISCUSSION THREAD

**Discussion by Blast06** [codemagic-ci-cd/codemagic-docs#2193] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [x] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Error:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processReleaseGoogleServices'.
> com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 15844 path $

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2m 11s
Running Gradle task 'bundleRelease'...                            132.1s
Gradle task bundleRelease failed with exit code 1

Build failed :|

1. Clone this repo
2. Run a workflow to see the error
3. Done

### Expected results

The output of the .apk file

### Actual results

Not getting the apk file and it throws:

* What went wrong:
Execution failed for task ':app:processReleaseGoogleServices'.
> com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 1 column 15844 path $

### Build id (optional)

654eb54ebf42fa857dd38d72

**Comment by himesh-cm** [comment]
Hi @Blast06, Can you please re-check the `JSON` file, which you uploaded in the Google Play section, if the formatting is alright?

**Comment by himesh-cm** [comment]
Hi @Blast06, I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
